test/access-control.integration.js

```diff
@@ -163,7 +169,7 @@ describe('access control - integration', function() {
     }
   });
 
-  describe('/accounts', function() {
+  describe('/accounts with replaceOnPUT true', function() {
```

Comment:
Modifying existing test suite and adding more tests into it is not a good practice. It makes it more difficult to understand the scope of this change, why are we changing existing tests not related to replace* methods to run against a model with `replaceOnPUT` set to true?

A better approach is to create a new `describe` block where you will use the new `accountWithReplaceOnPUTtrue` and then add only tests that are related to `replaceOnPUT` into this new block.